cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -100,5 +100,7 @@ export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
 
         export SONOBUOY_EXTRA_ARGS="--plugin systemd-logs --plugin e2e"
         hack/conformance.sh $conformance_config
+
+        KUBEVIRT_E2E_FOCUS=istio hack/functests.sh
```

Comment:
@dhiller do we have access to the kubevirt repo from this one ? Is there any easy way to run those tests ?

Reply:
Perfect, a lot better than my plan A (... which was to clone the kubevirt repo / grab the test binary from the web, and execute it w/ the `focus` on istio. 

Makes sense to me @ormergi . Can you push to this PR ?

Reply:
I posted a draft to test this PR changes with kubevirt istio tests
https://github.com/kubevirt/kubevirtci/pull/919